### DISCUSSION TITLE
Low power timers are missing in PeripheralPins.c

### DISCUSSION THREAD

**Discussion by PerennialNovice** [stm32duino/Arduino_Core_STM32#2331] [Q&A] [answered]
I just checked and found that in none of the PeripheralPins.c files the PinMap_TIM[] contains the LPTIMx pins

**Comment by fpistm** [comment, marked as answer]
Hi @PerennialNovice 
It is not an issue, the LPTIM are not supported by the core that's why they are not in the PinMap_TIM[] .
Anyway you can use it using the LL or HAL drivers.
